#initially based on debian's patch: https://sources.debian.net/patches/texstudio/2.11.2+debian-1/
#modification: in Settings menu: disable whole groupBox instead of all single elements inside it
--- configdialog.ui	2016-10-11 22:02:43.000000000 +0200
+++ configdialog.ui	2016-11-02 13:40:22.780668671 +0100
@@ -507,6 +507,9 @@
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_Update">
+             <property name="enabled">
+              <bool>false</bool>
+             </property>
              <property name="title">
               <string>Update</string>
              </property>

--- configmanager.cpp	2016-10-11 22:02:43.000000000 +0200
+++ configmanager.cpp	2016-11-09 12:56:50.392812690 +0100
@@ -472,7 +472,7 @@
 	registerOption("Thesaurus/Database", &thesaurus_database, "<dic not found>", &pseudoDialog->comboBoxThesaurusFileName);
 
 	//updates
-	registerOption("Update/AutoCheck", &autoUpdateCheck, true, &pseudoDialog->checkBoxAutoUpdateCheck);
+	registerOption("Update/AutoCheck", &autoUpdateCheck, false, &pseudoDialog->checkBoxAutoUpdateCheck);
 	registerOption("Update/UpdateLevel", &updateLevel, 0, &pseudoDialog->comboBoxUpdateLevel);
 	registerOption("Update/AutoCheckInvervalDays", &autoUpdateCheckIntervalDays, 7, &pseudoDialog->spinBoxAutoUpdateCheckIntervalDays);
 	registerOption("Update/LastCheck", &lastUpdateCheck, QDateTime());
